plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/JobDslPlugin.groovy

```diff
@@ -2,35 +2,79 @@ package com.here.gradle.plugins.jobdsl
 
 import com.here.gradle.plugins.jobdsl.tasks.GenerateXmlTask
 import com.here.gradle.plugins.jobdsl.tasks.UpdateJenkinsTask
+
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.tasks.Copy
 
 /**
  * Gradle plugin configuration for the Job DSL plugin. Configures tasks, extensions, and source sets.
  */
 class JobDslPlugin implements Plugin<Project> {
 
     @Override
+    @SuppressWarnings('DuplicateMapLiteral')
     void apply(Project project) {
         project.extensions.create('jobdsl', JobDslPluginExtension, project)
 
-        def execTasks = []
-        execTasks += project.task('dslGenerateXml', type: GenerateXmlTask)
-        execTasks += project.task('dslUpdateJenkins', type: UpdateJenkinsTask)
-
         project.plugins.apply('groovy')
 
-        execTasks.each { task ->
-            task.dependsOn 'classes'
-        }
-
         project.sourceSets {
             jobdsl {
                 groovy {
                     srcDirs 'src/jobdsl'
                 }
             }
         }
+
+        project.configurations {
+            jenkinsPlugins {
+                description 'Jenkins plugins that are required to process the Job DSL scripts.'
+            }
+        }
+
+        project.repositories {
+            mavenCentral()
+
+            maven {
+                url 'https://repo.jenkins-ci.org/releases/'
+            }
+        }
+
+        project.dependencies {
+            jenkinsPlugins('org.jenkins-ci.plugins:job-dsl:1.66') {
+                exclude(module: 'groovy-all')
+            }
+            jenkinsPlugins('org.jenkins-ci.plugins:job-dsl:1.66@jar') {
+                exclude(module: 'groovy-all')
+            }
+            jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.9@jar'
+        }
+
+        project.task('resolveJenkinsPlugins', type: Copy) {
+            from project.configurations.jenkinsPlugins
+            into new File(project.sourceSets.main.output.resourcesDir, 'test-dependencies')
+            include '*.hpi'
+            include '*.jpi'
```

Comment:
You may want to combine with the line above by using `includes` (plural).

Reply:
Done.

Reply:
Didn't work, failed with `Could not find method includes()...`. Weird, maybe I used the wrong syntax.

Reply:
Turns out is should still be named `include` (singular), but take a comma separated list. See the `exclude `example at https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Copy.html

Reply:
Trying this syntax now.

Reply:
Still approving this now in case you'd like to merge anyway.